py/embedded-server/java-runtime/build.gradle

```diff
@@ -21,6 +21,10 @@ dependencies {
     runtimeOnly project(':logback-print-stream-globals')
     runtimeOnly project(':logback-logbuffer')
     Classpaths.inheritLogbackClassic(project)
+
+    if (!hasProperty('excludeHotspotImpl')) {
```

Comment:
this might not be intuitive, but probably worth having

Reply:
This is how the server apps build up their classpath... hotspot-impl was previously missing from the embedded classpath, even though we are adding the jvm flags appropriately.

Reply:
Right, I think they "conflicted" when they were added, so it didn't get put here. My remark is mostly about the fact that no user will have the chance to remove this dependency, the flag is effectively only good for local debugging, since you would need to produce and install a custom wheel.